plugin/trino-delta-lake/pom.xml

```diff
@@ -416,6 +416,7 @@
                                 <exclude>**/TestDeltaLakeTableWithCustomLocationUsingGlueMetastore.java</exclude>
                                 <exclude>**/TestDeltaLakeRenameToWithGlueMetastore.java</exclude>
                                 <exclude>**/TestDeltaLakeGcsConnectorSmokeTest.java</exclude>
+                                <exclude>**/TestDeltaLakeRegisterTableProcedureUsingGlueMetastore.java</exclude>
```

Comment:
Let's move to before `TestDeltaLakeGcsConnectorSmokeTest` for grouping by Glue tests.

Reply:
Updated!